src/test/java/api/cart/PostCartTests.java

```diff
@@ -37,4 +38,13 @@ public void addCartBillingAddress() {
         Cart postCartBillingAddress = cartApiWrapper.postCartBillingAddress(billingAddress, postCartResponse.getId());
         Assert.assertEquals(cartApiWrapper.statusCode, HttpStatus.SC_OK, "incorrect error code");
     }
+
+    @Test
+    public void addCartShippingAddress() {
+        Cart cart = new Cart();
+        Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
+        ShippingAddress shippingAddress = CartUtils.addShippingAddressToCart("John", "Nashville", "TN", "US", "598780", "996325698", "shipping", "Lettuce 99", "No. 999");
+        Cart postCartShippingAddress = cartApiWrapper.postCartShippingAddress(shippingAddress, postCartResponse.getId());
+        Assert.assertEquals(cartApiWrapper.statusCode, HttpStatus.SC_OK, "incorrect error code");
```

Comment:
"incorrect error code" should be more specific: "Incorrect status code when adding shipping address". Please note that this is not an error code, is a response status code. Also, check if the address fields are returned correctly